servers/zts/schema/zts_server.sql

```diff
@@ -35,8 +35,9 @@ CREATE TABLE IF NOT EXISTS `zts_store`.`certificates` (
   `lastNotifiedServer` VARCHAR(512) NULL,
   `expiryTime` DATETIME(3) NULL,
   `hostName` VARCHAR(512) NULL,
+  `siaProvider` VARCHAR(256) NULL,
   PRIMARY KEY (`provider`, `instanceId`, `service`),
-  INDEX `idx_hostName` (`hostName` ASC))
+  INDEX `idx_hostName` (`hostName` ASC) VISIBLE)
```

Comment:
how did we get the VISIBLE flag here?

Reply:
It is mandatory for current version of `MySQL Workbench`

Reply:
So who ever touch a table via `MySQL Workbench`, it change the index with default value `VISIBLE`. 
Since that is the default value, it should be fine as I don't think Athenz uses any indexes not to be used by optimizer.